src/elements/hero-block.html

```diff
@@ -0,0 +1,156 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+
+<link rel="import" href="../../bower_components/app-layout/app-toolbar/app-toolbar.html">
+
+<link rel="import" href="../../bower_components/iron-pages/iron-pages.html">
+<link rel="import" href="../../bower_components/iron-icon/iron-icon.html">
+
+<link rel="import" href="../../bower_components/plastic-image/plastic-image.html">
+
+<link rel="import" href="../mixins/redux-mixin.html">
+<link rel="import" href="./hoverboard-icons.html">
+<link rel="import" href="./shared-styles.html">
+
+<dom-module id="hero-block">
+  <template>
+    <style is="custom-style" include="shared-styles flex flex-alignment positioning"></style>
+
+    <style>
+
+      :host {
+        display: block;
+      }
+
+      .hero-block {
+        padding: 0;
+        height: auto;
+        color: var(--hero-color);
+      }
+
+      .hero-overlay {
+        background-color: rgba(0, 0, 0, 0.6);
+        opacity: 0;
+        transition: opacity 0.3s;
+      }
+
+      .hero-overlay[show] {
+        opacity: 1;
+      }
+
+      .hero-image {
+        background-color: var(--hero-background-color);
+        transition: background-color 0.3s;
+      }
+
+      .container {
+        width: 100%;
+      }
+
+      .hero-content {
+        transform: translateZ(0);
+      }
+
+      .home-content {
+        max-width: 480px;
+      }
+
+      .hero-logo {
+        --iron-image-height: 48px;
+        margin-top: 30px;
+      }
+
+      .hero-title {
+        margin: 30px 0;
+        font-size: 40px;
+        text-transform: capitalize;
+      }
+
+      .info-items {
+        margin-top: 8px;
+      }
+
+      .info-item:not(:last-of-type) {
+        margin-right: 8px;
+      }
+
+      .hero-description {
+        margin-bottom: 30px;
+        max-width: 600px;
+      }
+
+      @media (min-width: 640px) {
+        .hero-block {
+          height: initial;
+        }
+
+        .container {
+          padding-top: 120px;
+          padding-bottom: 60px;
+        }
+
+        .hero-logo {
+          --iron-image-height: 70px;
+          margin-top: 0;
+        }
+      }
+
+    </style>
+
+    <app-toolbar class="hero-block" layout start vertical center-justified>
+      <plastic-image
+        class="hero-image"
+        srcset="[[hero.background.image]]"
```

Comment:
srcset$ to avoid problems on safari

Reply:
no, it's polymer property

Reply:
but sometimes you use it, and sometimes you don't

http://take.ms/m9yNO

Reply:
or you mean srcset is a property and we don't need this, hidden is an attribute, correct?

Reply:
Because sometimes we need to have attribute in a DOM, sometimes just pass data to the polymer element

Reply:
got it